code_generator/End2End/GeneratedDependencies/Generated/improbable/testschema/ExhaustiveBlittableSingularTranslation.cs

```diff
@@ -27,20 +27,21 @@ public class Translation : ComponentTranslation, IDispatcherCallbacks<global::Im
             public override ComponentType[] CleanUpComponentTypes => cleanUpComponentTypes;
             private static readonly ComponentType[] cleanUpComponentTypes = 
             { 
-                typeof(ComponentsUpdated<SpatialOSExhaustiveBlittableSingular>), typeof(AuthoritiesChanged<SpatialOSExhaustiveBlittableSingular>),
+                typeof(AuthoritiesChanged<SpatialOSExhaustiveBlittableSingular>),
+                typeof(ComponentsUpdated<SpatialOSExhaustiveBlittableSingular.Update>), 
             };
 
 
-            private static readonly ComponentPool<ComponentsUpdated<SpatialOSExhaustiveBlittableSingular>> UpdatesPool =
-                new ComponentPool<ComponentsUpdated<SpatialOSExhaustiveBlittableSingular>>(
-                    () => new ComponentsUpdated<SpatialOSExhaustiveBlittableSingular>(),
-                    (component) => component.Buffer.Clear());
-
             private static readonly ComponentPool<AuthoritiesChanged<SpatialOSExhaustiveBlittableSingular>> AuthsPool =
                 new ComponentPool<AuthoritiesChanged<SpatialOSExhaustiveBlittableSingular>>(
                     () => new AuthoritiesChanged<SpatialOSExhaustiveBlittableSingular>(),
                     (component) => component.Buffer.Clear());
 
+            private static readonly ComponentPool<ComponentsUpdated<SpatialOSExhaustiveBlittableSingular.Update>> UpdatesPool =
```

Comment:
Why has this moved around?

Reply:
In out current implementation, each component is guaranteed to have authority changed messages but will only have componentsUpdated messages if defined component fields. The change in the ordering is to reflect the change in importance.